server/src/main/java/org/activityinfo/ui/client/util/GwtUtil.java

```diff
@@ -135,4 +139,15 @@ public static Rectangle getBsContainerRectangle(com.google.gwt.user.client.Eleme
         }
         return null;
     }
+
+    public static void removeChildren(ComplexPanel panel) {
```

Comment:
Thanks! Removed redundant code in https://github.com/bedatadriven/activityinfo/commit/4e23656ef4f1b3da8c3dd7d51d55c7fe87a915ef